README.md

```diff
@@ -69,3 +69,17 @@ $ forge --help
 $ anvil --help
 $ cast --help
 ```
+
+## Validate Upgrade Safety 
+1. Checkout to the deployed implementation commit 
+2. Copy the YieldDistributor.sol to test/upgrades/<version>/YieldDistributor.sol
```

Comment:
this needs to have the path wrapped in backquotes

Reply:
imo the line should be:

Copy "YieldDistributor.sol" to `test/upgrades/<version>/YieldDistributor.sol`

Reply:
addressed with a6b0ac2